OrbitCore/CaptureData.h

```diff
@@ -125,11 +125,19 @@ class CaptureData {
     return tracepoint_info_manager_->Get(key);
   }
 
+  [[nodiscard]] const TracepointInfoManager* GetTracepointInfoManager() const {
+    return tracepoint_info_manager_.get();
+  };
+
   [[nodiscard]] const std::map<uint64_t, orbit_client_protos::TracepointEventInfo>&
   GetTracepointsOfThread(int32_t thread_id) const {
     return tracepoint_event_buffer_->GetTracepointsOfThread(thread_id);
   }
 
+  [[nodiscard]] const std::deque<orbit_client_protos::TracepointEventInfo> GetTracepointEvents()
+      const {
+    return tracepoint_event_buffer_->GetTracepointEvents();
```

Comment:
From a caller perspective it is not clear that this returns a copy, so it can not be modified by a different thread. 
So I would always need to double-check, if it is safe to use that method. I'd rather introduce a `ForEachTracepointEvent` method, of which it is more obvious, that I am safe to use.

Reply:
Thank you so much for the advice! I will update the code to have a 'ForEachTracepointEvent' method.